### PR TITLE
PIM-709

### PR DESCRIPTION
**Pull request by mfullmore**
OK not totally sure this is what needs to be done but it's a start.

I need an endpoint to create a Signed URL for our digital asset details page, and list page. I think this is how I should go about adding this endpoint to the CloudFileStorage app, but I am open for suggestions.

Jest tests are coming

_But how do I test this endpoint locally? That wasn't clear to me._  I was able to figure this out, and added local build support for this last Commit. Please review and provide feedback.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,13 +10,18 @@
     "scripts": {
         "start": "node build/v1/main.js PRODUCTION",
         "dev": "node build/v1/main.js",
-        "postinstall": "tsc && rm -v -rf src",
+        "postinstall": "npx tsc",
```
Comment (mfullmore) [inline package.json:13]: @clawyq the `rm` was causing issues locally and I am not sure it's really needed on the production host. Just wanted to call this to your attention.

Comment (clawyq) [reply package.json:13]: its included so the source files dont sit on the heroku app. there isn't a need for the .ts files anymore after they have been transpiled to .js.

Comment (mfullmore) [reply package.json:13]: Is there another way to delete the files on the server and not locally? if your run `npm install` locally which you should the `postinstall` script runs automatically.

Comment (mfullmore) [reply package.json:13]: @clawyq so the trade off of deleting some files on the server vs having a local dev environment is a trade off I am willing to take on. Please re-review.

Comment (mfullmore) [reply package.json:13]: Thoughts? @clawyq

**Comment by mfullmore** [comment]
Adding @tmatthews092 to review my local environment setup.

**Comment by mfullmore** [comment]
@clawyq please review this let me know if you are ok with this change. It is need to run locally.

**Review by clawyq — APPROVED** [review]

**Pull request merged**